bridgesync/downloader.go

```diff
@@ -102,6 +108,13 @@ func buildBridgeEventHandler(contract *polygonzkevmbridgev2.Polygonzkevmbridgev2
 			return fmt.Errorf("failed to extract bridge event calldata (tx hash: %s): %w", l.TxHash, err)
 		}
 
+		isNativeToken := false
+
+		if gasTokenAddress == bridgeEvent.OriginAddress ||
+			bridgeEvent.OriginAddress == common.Address(aggkitCommon.ZeroHash.Bytes()) {
```

Comment:
Although it works this way as well, having the variable of type `common.Address` that corresponds to zero address would be nicer IMO (from a semantical point of view it is incorrect).